packages/webpack/src/config/client.js

```diff
@@ -124,6 +125,12 @@ export default class WebpackClientConfig extends WebpackBaseConfig {
       }))
     }
 
+    if (this.options.build.crossorigin) {
+      plugins.push(new CorsPlugin({
```

Comment:
Could be require the plugin lazily here (Would that make any difference)?

Reply:
This part is for building, so it won't be bundled into server.js, so I thinks it's fine.